### DISCUSSION TITLE
ward and room information of patients

### DISCUSSION THREAD

**Discussion by yairdgani** [MIT-LCP/mimic-code#1143] [MIMIC-General]
Hi,

Is there a specific ward and room information of a patient?
I see there is a ward description (Medicine, Vascular etc.) but is there information of the specific ward, meaning, for example: Medicine Ward no. 3, room no. 18?

Thanks in advance

**Comment by briangow** [comment]
In MIMIC-III you will find `ward_id` variables which provide information about the physical location within the hospital. See the transfers table documentation on the website for more information: https://mimic.mit.edu/docs/iii/tables/transfers/#prev_wardid-curr_wardid . This `ward_id` isn't necessarily indicative of the care being provided though. Over time the hospital may shift the type of care being provided in a given physical location. Also, depending upon bed availability, the type of care being provided may vary in a given physical location.  

Given that physical location is not typically of importance in research we chose to not provide `ward_id` in MIMIC-IV. 

In most cases using `careunit` is preferable to using `ward_id`.

**Reply by yairdgani** [reply]
Thank you.
I will try to be more specific:
I want to research the relation between patients.
For this matter I need the physical locations and the relevant time.
Regarding physical location, I understand from that there is ward id in mimic3, but, is there room or bed id as well?
Regarding time, is there a way to sync the time between 2 patients (at least from the same ward  or room)?

**Comment by briangow** [comment]
The website explains how dates and times work (see [About/Concepts/](https://mimic.mit.edu/docs/iv/about/concepts/#date-shifting)). After date shifting, actual transpired times are derivable for an individual patient. However, since the date shifting is random at the patient level I don't think you'll be able to determine when multiple patients have overlapping stays.
